src/server/rewrites.js

```diff
@@ -0,0 +1,64 @@
+var fs = require('fs');
+var logger = require('./logging').logger;
+var PathService = require('../services/path');
+
+
+/**
+ * @todo This same logic of reading JSON files is in a few places. It should be
+ *       abstracted into something like `ConfigService.readConfigFile(fileName)`
+ */
+var REWRITES_FILE = 'rewrites.json';
+var SITE_KEY = 'developer.rackspace.com';
+
+try {
+    var rewrites = JSON.parse(fs.readFileSync(
+        PathService.getConfigPath(REWRITES_FILE),
+        'utf-8'
+    ))[SITE_KEY];
```

Comment:
Ah. The intent of having multiple sites within the same content map is that you can host multiple properties (developer.rackspace.com, docs.rackspace.com, whatever) on the _same_ deconst instance, just by pointing the DNS records for each to the same presenter CLB. To support this, we were pulling the domain from the presented URL and using that as an index into the content map at request-time, rather than once globally. For development (or other cases where you won't have DNS set up properly), if the env var `PRESENTED_URL_DOMAIN` is nonempty, that was used instead.

With that being said, we weren't actually _using_ this yet, so I don't think it's a barrier to :ship:.